plaso/output/shared_elastic.py

```diff
@@ -110,9 +110,15 @@ def _FlushEvents(self):
       # pylint does not recognizes request_timeout as a valid kwarg. According
       # to http://elasticsearch-py.readthedocs.io/en/master/api.html#timeout
       # it should be supported.
-      self._client.bulk(
-          body=self._event_documents, doc_type=self._document_type,
-          index=self._index_name, request_timeout=self._DEFAULT_REQUEST_TIMEOUT)
+      bulk_args = dict(
+        body=self._event_documents, index=self._index_name,
+        request_timeout=self._DEFAULT_REQUEST_TIMEOUT)
+
+      # TODO: Remove once Elasticsearch v6.x is deprecated.
+      if self.GetClientMajorVersion() < 7:
```

Comment:
I opt to store the major version in a variable instead of calling GetClientMajorVersion multiple times

Reply:
nvm